server/sonar-web/src/main/js/apps/projects/filters/Filter.js

```diff
@@ -40,19 +40,25 @@ export default class Filter extends React.PureComponent {
     getFacetValueForOption: React.PropTypes.func,
 
     halfWidth: React.PropTypes.bool,
-    highlightUnder: React.PropTypes.func
+    highlightUnder: React.PropTypes.any
```

Comment:
I'm not comfortable with `any`. It's hard to tell what the component expects as an input. Can you stick to a function please?